mozilla/foundation.mozilla.org
# Added missing privacy policy links

Closes #7822 
Related PRs/issues #

**Link to sample test page**: https://foundation-s-privacy-no-h6xgje.mofostaging.net/en/privacynotincluded/important-house-many/

**Steps to test:**

1. Please visit the test page linked above
2. Scroll down to "Links to privacy policy information"
3. If everything looks as expected, testing is complete!
